quicklogic/common/cmake/quicklogic_install.cmake

```diff
@@ -56,6 +56,10 @@ function(DEFINE_QL_TOOLCHAIN_TARGET)
   install(FILES ${VPR_CONFIG}
           DESTINATION share/symbiflow/scripts/${FAMILY})
 
+  # Example design to run through the flow
+  install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/quicklogic/${FAMILY}/tests/counter_16bit/*.v
+	  DESTINATION share/symbiflow/tests/counter_16bit
+  
```

Comment:
There is a syntax error here (as reported by the CI). You're missing the closing `)` bracket.

Reply:
fixed